RM_AppleStore/RM_AppleStore/Controllers/OnboardingPageViewController.swift

```diff
@@ -0,0 +1,160 @@
+//
+
+import UIKit
+
+/// Управление онбордингом и его контентом
+final class OnboardingPageViewController: UIPageViewController {
+    // swiftlint: disable all
+    // MARK: - Constants
+    private enum Constants {
+        static let firstOnboardingImageName = "firstImage"
+        static let secondOnboardingImageName = "secondImage"
+        static let thirdOnboardingImageName = "thirdImage"
+        static let skipButtonTitle = "SKIP"
+        static let nextButtonTitle = "NEXT"
+        static let getStartedButtonTitle = "GET STARTED!"
+        static let loginDefaultsKey = "Was login?"
+        static let firstOnboardingTitle = "Track Your Cycle"
+        static let firstOnboardingDescription = "Manage irregular period and learn how to improve your period"
+        static let secondOnboardingTitle = "Plan Your Pregnancy"
+        static let secondOnboardingDescription = "Favorable days are important. Vestibulum rutrum quam vitae frengila tincidunt."
+        static let thirdOnboardingTitle = "Daily Health Insight"
+        static let thirdOnboardingDescription = "Personal health insight. Vestibulum rutrum quam vitae frengila tincidunt."
+    }
+    // swiftlint: enable all
+    
+    // MARK: - Private Visual Components
+    private let onboardingImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }
+    
+    private lazy var skipButton = {
+        let button = UIButton(frame: CGRect(x: 40, y: view.frame.height - 66, width: 100, height: 30))
+        button.setTitle(Constants.skipButtonTitle, for: .normal)
+        button.setTitleColor(.darkGray, for: .normal)
+        button.addTarget(self, action: #selector(finishOnboardingAction), for: .touchUpInside)
+        return button
+    }()
+    
+    private lazy var nextButton = {
+        let button = UIButton(frame: CGRect(x: 240, y: view.frame.height - 66, width: 100, height: 30))
+        button.setTitle(Constants.nextButtonTitle, for: .normal)
+        button.setTitleColor(.tintColor, for: .normal)
+        button.addTarget(self, action: #selector(nextButtonAction), for: .touchUpInside)
+        return button
+    }()
+    
+    private lazy var getStartedButton = {
+        let button = UIButton(frame: CGRect(x: 120, y: view.frame.height - 66, width: 150, height: 50))
+        button.setTitle(Constants.getStartedButtonTitle, for: .normal)
+        button.setTitleColor(.tintColor, for: .normal)
+        button.isHidden = true
+        button.addTarget(self, action: #selector(finishOnboardingAction), for: .touchUpInside)
+        return button
+    }()
+    
+    // MARK: - Private properties
+    // swiftlint: disable all
+    private let onboardingViewControllers = [
+        OnboardingViewController(nibName: nil, bundle: nil, onboardingImageName: Constants.firstOnboardingImageName, title: Constants.firstOnboardingTitle, description: Constants.firstOnboardingDescription),
```

Comment:
```suggestion
        OnboardingViewController(
           nibName: nil,
           bundle: nil,
            onboardingImageName: Constants.firstOnboardingImageName,
           title: Constants.firstOnboardingTitle,
            description: Constants.firstOnboardingDescription
            ),
```
либо в метод и 3 раза вызывать его